prow/scripts/skr-svcat-migration.bash

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+# Script to run SKR svcat migration
+
+set -euo pipefail
+make -C /home/prow/go/src/github.com/kyma-project/kyma/tests/fast-integration ci-skr-svcat-migration
```

Comment:
This script is unnecessary. You can remove it in favour of defining command directly in the ProwJob template.